src/bitwarden_exporter/__main__.py

```
@@ -1,283 +1,138 @@
+        False, help="Allow duplicates entries in export, In bitwarden each item can be in multiple collections,"
+    ),
+    tmp_dir: str = typer.Option(
+        default=tempfile.mkdtemp(prefix="bitwarden_exporter_"),
+        help="Temporary directory to store temporary sensitive files.",
+        show_default="Temporary directory",
+    ),
+    bw_executable: str = typer.Option(
+        "bw",
+        help="Path to the Bitwarden CLI executable.",
+    ),
+    debug: bool = typer.Option(
```

Ruff (0.14.1)

Boolean-typed positional argument in function definition

(FBT001)